specification/MDF.tex

```diff
@@ -280,24 +280,45 @@ \subsection{Acquisition Parameters (group: \inlvar{/acquisition/}, non-optional)
+In MPI, a frame groups all data used to reconstruct a single MPI image/tomogram. In the simplest scenario, this data is acquired during one \inlvar{/drivefield/period}. The acquisition time increases by a factor of \inlvar{numAverages} if averaging is applied and by a factor of \inlvar{numPeriods} if data is recorded over several (averaged) cycles. 
+%In a multi-patch setting $J$ \inlvar{offsetField}s or mechanical movements shift the gradient field by \inlvar{offsetFieldShift}, where $J$ is the number of patches \inlvar{numPatches} of a multi-patch measurement. For each shift at least one full drive field cycle is used to acquire the measurement data. %For instance, a Cartesian 2D trajectory with 100 lines could be realized by setting \mbox{\inlvar{numPatches} $ = 100$}.
+
+\textcolor{red}{TODO: Discuss the dimension $Y$. It is usually equal to $J$ when considering a static multi-patch scenario. But it can also be finer resolved to describe continuous FF.}
 
 \end{multicols}
 
 \noindent \begin{tabularx}{\columnwidth}{lllllX}
 \textbf{Parameter} & \textbf{Type} & \textbf{Dim} & \textbf{Unit/Format} & \textbf{Optional} & \textbf{Description} \\ \hline
 \inlvar{startTime} & \inltab{String} & 1 & yyyy-mm-ddThh:mm:ss.ms & no & UTC start time of MPI measurement \\ \hline
-\inlvar{framePeriod} & \inltab{Float64} & 1 & s & no & Time to complete a full frame (product of \inlvar{/drivefield/period}, \inlvar{numPeriods}, \inlvar{numAverages}, and \inlvar{numPatches}) \\ \hline
-\inlvar{numPeriods} & \inltab{Int64} & 1 & 1 & no & Number of drive-field periods per patch \\ \hline
-\inlvar{numAverages} & \inltab{Int64} & 1 & 1& no & Number of block averages per patch\\ \hline
-\inlvar{numPatches} & \inltab{Int64} & 1 & 1 & no & Number of patches within a frame denoted by $J$ \\ \hline
+\inlvar{framePeriod} & \inltab{Float64} & 1 & s & no & Time to complete a full frame (product of \inlvar{/drivefield/period}, \inlvar{numPeriods}, and \inlvar{numAverages} \\ \hline
+\inlvar{numPeriods} & \inltab{Int64} & 1 & 1 & no & Number of drive-field periods within a frame denoted by $J$ \\ \hline
+\inlvar{numAverages} & \inltab{Int64} & 1 & 1& no & Number of block averages per drive-field period \\ \hline
 \inlvar{numFrames} & \inltab{Int64} & 1 & 1& no & Number of available measurement frames $N$ \\ \hline
-\inlvar{gradient} & \inltab{Float64} & $J \times 3$ & Tm$^{-1}\mu_0^{-1}$ & yes & Gradient strength of the selection field in $x$, $y$, and $z$ directions \\ \hline
-\inlvar{offsetField} & \inltab{Float64} & $J \times 3$ & T$\mu_0^{-1}$ & yes & Offset field applied for each patch in the measurement sequence \\ \hline
-\inlvar{offsetFieldShift} & \inltab{Float64} & $J \times 3$ & m & yes & Position of the field free point (relative to origin/center) \\ \hline
+\inlvar{gradient} & \inltab{Float64} & $Y \times 3$ & Tm$^{-1}\mu_0^{-1}$ & yes & Gradient strength of the selection field in $x$, $y$, and $z$ directions \\ \hline
```

Comment:
Do we really want to have one Y?

Is it not better to split into Y_g (gradient) and Y_o (offset)? Maybe we want to manipulate both at different time scales.

Reply:
could make sense. Although this gives a lot a freedom. In that case one could also have Y_d for the drive field.

Reply:
but the drive field can be handled via the custom wave form

Reply:
In my opinion all this low frequency stuff should only affect the signal encoding and not the signal generation in MPI. Good question, how a low freq modulation of df amplitude should be handled. For low enough frequencies this could work.

Reply:
I remove the DF custom waveform since this was kind of ambiguous.

Reply:
I see

Reply:
It was not only ambiguous but very inflexible. You either had to use the custom waveform for all channels or for non.

Reply:
But back to topic: We could also not introduce `Y_g` and `Y_o`. This is a little bit more strict and the worst that can happen is that the amount of metadata is a little higher. If we really feel that this is an issue, we can introduce `Y_g` and `Y_o` without breaking the spec. The case `Y_g` = `Y_o` would also be handled.

Reply:
In this case lets stick with one `Y`.